### DISCUSSION TITLE
Missing "Order By" in "MyWork"->"Repositories"

### DISCUSSION THREAD

**Discussion by carry0987** [community/community#75347] [Mobile]
The Repositories in sidebar doesn't have OrderBy option, and the order is different from Profile->Repositories

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.137.2 (114881244)
Device: iPad Air 5 (iOS 17.1.1)
</details>

**Comment by HenSquared** [comment]
Hi @carry0987 , 

Thanks a lot for the feedback. I've opened an Issue to track this feature request and discuss with the team internally. If we have more updates, we will update this thread or update the release notes.
